New log file opened at Mon Nov 16 20:05:45 CET 2015 by 9240@Obsidian
Taking next connection to: 56684
New client attempting to connect on port: 56685-56684
  Address = /192.168.178.14:56684
This client on port 56685 has uuid 0a709ff1-4f7f-4ef7-b95b-f50a5dc9b1b8 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-0a709ff1-4f7f-4ef7-b95b-f50a5dc9b1b8
Taking next connection to: 56684
New client attempting to connect on port: 56779-56684
  Address = /192.168.178.14:56684
Handshake exception on socket: 56779
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 56684
New client attempting to connect on port: 56780-56684
  Address = /192.168.178.14:56684
This client on port 56780 has uuid 85dc529d-a831-4963-85f3-78055bf159d4 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-0a709ff1-4f7f-4ef7-b95b-f50a5dc9b1b8, play-fork-run-85dc529d-a831-4963-85f3-78055bf159d4
Taking next connection to: 56684
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 56684
